fhir-model/src/main/java/com/ibm/fhir/model/path/util/FHIRPathUtil.java

```diff
@@ -257,12 +358,12 @@ public static Temporal getTemporal(TemporalAccessor temporalAccessor) {
             return yearMonth.atDay(1);
         } else if (temporalAccessor instanceof LocalDate) {
             return (LocalDate) temporalAccessor;
+        } else if (temporalAccessor instanceof LocalDateTime) {
+            return (LocalDateTime) temporalAccessor;
         } else if (temporalAccessor instanceof ZonedDateTime) {
             return (ZonedDateTime) temporalAccessor;
         } else if (temporalAccessor instanceof LocalTime) {
             return (LocalTime) temporalAccessor;
```

Comment:
This pattern is repeated in multiple spots in JDBCBuilder... only mentioning here that we should probably be consistent and use this as the reference point

Reply:
Some sort of Map<Class<?>, Function> might be cleaner. I'll leave it for now.

Reply:
No problem... just verbalizing the pattern